jbcli/jbcli/cli/jb.py

```diff
@@ -608,7 +608,7 @@ def freshstart(ctx, env, noupdate, noupgrade, ssh):
             subprocess.check_call(
                 ['killall', '-HUP' 'com.docker.hyperkit'])
             time.sleep(30)
-            start(noupdate=noupdate, ssh=ssh)
+            freshstart(env, noupdate=noupdate, noupgrade=noupgrade, ssh=ssh)
```

Comment:
this was a bug I noticed in `freshstart`. It tries to re-run itself when waiting for docker to restart, but it was still calling the old `start` instead of `freshstart`